operator/pkg/resources/testdata/Complex_struct_in_additional_configuration.golden

```diff
@@ -5,6 +5,9 @@ redpanda:
         - host:
             address: cluster-0.cluster.local
             port: 33145
+        - host:
+            address: cluster-first-0.cluster.local
+            port: 33145
```

Comment:
Why seed server list have more hosts if only single Node Pool is set? Shouldn't the old host (short name of the pod) be removed?

The `first` node pool is consider second node pool as the `default` is is not included in the `Spec.NodePools`, but it is implied from `Spec.Replicas`, right?

If it's not then what would be the name of the pods? I'm worry that startup will be longer due to first host DNS name being not available in the cluster.

Reply:
this is correct. Spec.Replicas drives the default nodepool "cluster" (with no suffix), and the nodepool provided is an additional STS.